src/hooks/useLocations.js

```diff
@@ -1,10 +1,11 @@
-import { useState, useEffect } from 'react';
+import { useState, useEffect, useRef } from 'react';
 
-export const useLocations = (lat, lng) => {
+export const useLocations = ({ lat, lng, isDragged }, interval) => {
   const [locations, setLocations] = useState([]);
   const [status, setStatus] = useState('idle');
   const [error, setError] = useState(null);
   const [isFetching, setIsFetching] = useState(false);
+  const ref = useRef(0);
```

Comment:
**Optional Suggestion:** In my humble opinion, `ref` variable is not clear. What kind of information should we store in this variable?. What do you think about those names `todayInMilliseconds`  or `nowInMilliseconds`?

Reply:
We renamed it to ``timeRef``

Reply:
Thanks, I think this name is a little more clear 👍 .